spring-cloud-starter-dataflow-server-cloudfoundry/pom.xml

```diff
@@ -17,11 +17,11 @@
 	<dependencyManagement>
 		<dependencies>
 			<!-- (re)Force reactor version -->
-			<dependency>
-				<groupId>io.projectreactor</groupId>
-				<artifactId>reactor-core</artifactId>
-				<version>${reactor.version}</version>
-			</dependency>
+			<!--<dependency>-->
+				<!--<groupId>io.projectreactor</groupId>-->
+				<!--<artifactId>reactor-core</artifactId>-->
+				<!--<version>${reactor.version}</version>-->
+			<!--</dependency>-->
```

Comment:
Ah crap, thought I *deleted* these.
Le 11 avr. 2016 6:23 PM, "Mark Pollack" <notifications@github.com> a écrit :

> In spring-cloud-starter-dataflow-server-cloudfoundry/pom.xml
> <https://github.com/spring-cloud/spring-cloud-dataflow-admin-cloudfoundry/pull/80#discussion_r59234766>
> :
>
> > @@ -17,11 +17,11 @@
> >  	<dependencyManagement>
> >  		<dependencies>
> >  			<!-- (re)Force reactor version -->
> > -			<dependency>
> > -				<groupId>io.projectreactor</groupId>
> > -				<artifactId>reactor-core</artifactId>
> > -				<version>${reactor.version}</version>
> > -			</dependency>
> > +			<!--<dependency>-->
> > +				<!--<groupId>io.projectreactor</groupId>-->
> > +				<!--<artifactId>reactor-core</artifactId>-->
> > +				<!--<version>${reactor.version}</version>-->
> > +			<!--</dependency>-->
>
> Any reason to add commented out sections?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-cloud/spring-cloud-dataflow-admin-cloudfoundry/pull/80/files/7152f96736f55af3106e8737f77439fab28229a6#r59234766>
>